packages/status/package.json

```diff
@@ -21,6 +21,7 @@
     "@heroku-cli/command": "^8.4.1",
     "@oclif/command": "^1.5.11",
     "@oclif/config": "^1.12.10",
+    "@oclif/core": "^1.20.4",
     "@oclif/errors": "^1.2.2",
     "cli-ux": "^4.9.3",
```

Comment:
oclif/command, oclif/config, oclif/errors, and cli-ux are the old oclif packages being replaced by core. Is there a reason they're still in here?

Reply:
Yeah, I completely forgot about removing them. They're gone now. Thank you.